SingularityService/src/main/java/com/hubspot/singularity/resources/HistoryResource.java

```diff
@@ -215,6 +238,22 @@ public SingularityDeployHistory getDeploy(@ApiParam("Request ID for deploy") @Pa
   }
 
   @GET
+  @Path("/request/{requestId}/deploys/count")
```

Comment:
For sure, we'll be deprecating the old one but still need both concurrently so we'll still need a way to differentiate.